models/scripts/PCA.py

```diff
@@ -0,0 +1,55 @@
+from tabpy_tools.client import Client
```

Comment:
I suggest we have docstring for all the new file. In this case, we can explain what is this model and where is the documentation for how to use it.